packages/rx_bloc_cli/mason_templates/bricks/lib_internationalisation/__brick__/lib/lib_internationalisation/data_storages/language_picker_shared_preferences_instance.dart

```diff
@@ -0,0 +1,17 @@
+{{> licence.dart }}
+
+import 'package:shared_preferences/shared_preferences.dart';
+
+class LanguagePickerSharedPreferencesInstance {
```

Comment:
From what I see this is not a language specific shared pref

Reply:
Yes, I have removed it